Lib/email/_header_value_parser.py

```diff
@@ -1101,13 +1101,14 @@ def get_unstructured(value):
             token, value = get_fws(value)
             unstructured.append(token)
             continue
+        encoded_format_invalid = False
         if value.startswith('=?'):
             try:
                 token, value = get_encoded_word(value)
-            except errors.HeaderParseError:
-                # XXX: Need to figure out how to register defects when
-                # appropriate here.
-                pass
+            except errors.HeaderParseError as e:
+                if "encoded word format invalid" in str(e):
+                    unstructured.defects.append(errors.InvalidHeaderDefect(str(e)))
```

Comment:
Invalid encoded word format is not necessarily a defect, it only means that we should proceed with other type of parsers on the value.

Reply:
Done